features/machine/OCP24363.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Machine taints
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  @destructive
+  Scenario Outline: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    Given I clone a machineset named "machineset-24363"
+    Then the step should succeed
+
+    Given I store the last provisioned machine in the :machine clipboard
+    And evaluation of `machine(cb.machine).node_name` is stored in the :nodeRef_name clipboard
```

Comment:
Let's use snake cases for all variables.

Reply:
ok